src/css/controls/imports/modal.less

```diff
@@ -0,0 +1,32 @@
+.jw-modal {
+    width: 284px;
+
+    .jw-breakpoint-7 &,
+    .jw-breakpoint-6 &,
+    .jw-breakpoint-5 & {
+        .rect(284px, 232px);
```

Comment:
No need to use rect since the width is already set on line 2; we can just specify the height here.